src/main/java/hwManifestGen/AllComponents.java

```diff
@@ -195,6 +235,276 @@ public static void setOptions(Options options) {
+			}
+   
+			}
+			else {
+			    // Second curl command
+			    String urlString1 = "https://" + system + command;
+			    URL url2 = new URL(urlString1);
+			    HttpsURLConnection con3 = (HttpsURLConnection) url2.openConnection();
+			    con3.setRequestMethod(type);
+			    con3.setRequestProperty("Content-Type", "application/json");
+			    con3.setRequestProperty("x-auth-token", authToken);
+			    con3.setDoOutput(true);
+			    String jsonInputString2 = "{\"CertificateCollection\":{\"@odata.id\":\"/redfish/v1/Managers/RMC/NetworkProtocol/HTTPS/Certificates\"}, \"Country\":\"US\", \"State\":\"California\", \"City\":\"Palo Alto\", \"Organization\":\"Hewlett Packard Enterprise Company\",\"OrganizationalUnit\":\"MCS\", \"CommonName\":\"h2-641-rmc.fchst.rdlabs.hpecorp.ne\"}";
+
+			    // Send JSON input string as request payload
+			    try (OutputStreamWriter out = new OutputStreamWriter(con3.getOutputStream())) {
+				out.write(jsonInputString2);
+			    }
+
+			    // Read the response from the connection
+			    BufferedReader in2 = new BufferedReader(new InputStreamReader(con3.getInputStream()));
+			    String inputLine2;
+			    StringBuffer content2 = new StringBuffer();
+			    while ((inputLine2 = in2.readLine()) != null) {
+				content2.append(inputLine2);
+			    }
+			    in2.close();
+
+			    // Convert the response to a string
+			    output = content2.toString();
+
+			    // Extract the CSRString using a regular expression
+			    Pattern pattern = Pattern.compile("\"CSRString\": \"(.*?)\"");
+			    Matcher matcher = pattern.matcher(output);
+			    if (matcher.find()) {
+				csrString = matcher.group(1);
+
+				// Print the second response line by line
+				String[] lines = csrString.split("\n");
+				for (String line : lines) {
+				    line = line.replaceAll("\\\\n", "\n");
+				    System.out.println("Certificate generated: " + line);
+				}
+
+			    // Store the file path
+				String file;
+
+			// Check if the "path" command option is present
+			if (cmd.hasOption("path")) {
+			    // Get the value of the "path" option
+			    String value = cmd.getOptionValue("path");
+			    System.out.println("Value of command option: " + value);
+
+			    // Create a File object with the specified directory path
+			    File directory = new File(value);
+
+			    // Check if the directory exists
+			    if (!directory.exists()) {
+				// Create the directory if it does not exist
+				boolean created = directory.mkdirs();
+				if (created) {
+				    System.out.println("Directory created: " + value);
+				} else {
+				    System.out.println("Failed to create directory: " + value);
+				}
+			    }
+
+			    // Set the file path as value + "output1.pem"
+			    file = value + "output1.pem";
+			} else {
+			    System.out.println("Output Path option not provided");
+			    // Set a default file path if "path" option is not provided
+			    String value = "/home/prerana/Desktop/PCVT/PCVT/src/main/java/hwManifestGen/";
```

Comment:
At the instructions the paths suggested were in general /opt/hpe/scl/ or /opt/hpe/scl/certificates for the outputs... so I suggest to modify this for /opt/hpe/scl/ or a directory whithin this path.

Reply:
Changed to the expected path